pkg/util/namespace/namespace_test.go

```diff
@@ -7,73 +7,58 @@ import (
 	"testing"
 )
 
-func TestIsOpenShift(t *testing.T) {
+func TestIsOpenShiftNamespace(t *testing.T) {
 	for _, tt := range []struct {
 		namespace string
 		want      bool
 	}{
 		{
-			want: true,
```

Comment:
I think we still need a test case for an empty namespace `""`.

Reply:
> It looks like we accidentally dropped a test case for empty string `""` instead of a namespace. We need it untill we implement [this suggestion](https://github.com/Azure/ARO-RP/pull/1994#discussion_r864939378).
> 
> Other than that - looks good to me.

Added a test case for the empty namespace `""`
> 
> @xiaoyu74 could you please create a work item to update geneva monitors before we merge this PR? I want to make sure that we don't forget that.

I have created an ADO work item [14351421](https://msazure.visualstudio.com/AzureRedHatOpenShift/_boards/board/t/Firefly/Stories/?workitem=14351421) based on the discussions.